app/serializers/profile_serializer.rb

```diff
@@ -13,4 +14,7 @@ class ProfileSerializer
   attribute :compliant_host_count do |profile|
     profile.hosts.count { |host| profile.compliant?(host) }
   end
+  attribute :business_objective do |profile|
+    profile.business_objective.title
```

Comment:
To match the UI in which we have business objective that looks like a string field on a profile, the API as written allows updating business_objective as a string. We find or create the BO for the user. Traditional REST APIs would force users to first create a BO with a BO API, get the ID, then pass `business_objective_id` to the profiles API. I'm open to both methods, but the way it's written is much more user friendly; I wouldn't probably do this if a profile had many BOs, but since it only has one, this seemed OK.